.github/scripts/build.sh

```diff
@@ -3,9 +3,8 @@
 set -e # exit on error
 set -x # echo on
 
-export TARGET=${TARGET:-aarch64-w64-mingw32}
-export CRT=${CRT:-msvcrt}
```

Comment:
export is not needed here anymore?

Reply:
It was needed when this value shall be propagated to other build scripts. Since now they are impring the variables through source, the export is no longer needed.